### ISSUE TITLE
NVDA can not read Accessibility View when screen mode not turned on

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#26070]
Testing -186218

* Turn off screen reader mode
* Go to Panel Chat
* NVDA reads "Press alt+f1 to open accessibility view"
* Press alt+f1
* NVDA reads the editor is not accessible at this time ... the editor has multiple content ... blank

IMHO this defeats some of the purpose of the accessible view, which should always be accessible (can be read by NVDA)
